extension/src/test/TaskRunner.test.ts

```diff
@@ -101,7 +101,7 @@ suite("Task Runner Tests", function () {
       {
         description: "Show release notes.",
         command: "update.showCurrentReleaseNotes",
-        openFile: "app.json",
+        openFile: "Xliff-test//app.json",
```

Comment:
I feel stupid... why the double `/`?

Reply:
Don't feel stupid!
I first had \\, but then I changed to forward slash because of you - and it just became a double one...

I'll clean this up...

Reply:
Well it says something of how highly I regard you :sweat_smile:

Reply:
I thought it was obvious that you should quit with that! 😜

And now I also see that GitHub needs \ to be escaped... One \ becomes \ , but also two \ becomes \\ , so I need four \ to write \\\\ 😣